docs/apache-airflow-providers-sftp/sftp.rst

```diff
@@ -0,0 +1,147 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+
+SFTPOperator
+==========================
+Use the :class:`~airflow.providers.sftp.operators.sftp.py` to
+transfer data between servers under sftp.
+
+Using the Operator
+------------------
+To start working with an operator, you need to register an SFTP \ SSH connection in Airflow Connections.
+Use ssh_conn_id to specify the name of the connection.
```

Comment:
Is this doc meant to describe both `SFTPOperator` and `SFTPBatchOperator` or just one of the operators? This sections reads like the doc is about `SFTPOperator` only but there are code snippets for `SFTPBatchOperator` as well further on.

Reply:
this doc for a both operators

Reply:
done, i separated docs